dangerfile.ts

```diff
@@ -24,6 +24,25 @@ const hasAddedLinesAfterVersionInChangelog = async (): Promise<boolean> => {
   return addedLines.some(line => line.ln > versionLineNumber)
 }
 
+const checkChangelogFormat = async () => {
+  const changelogDiff = await danger.git.structuredDiffForFile(CHANGELOG_FILE)
+  const addedLines = changelogDiff.chunks.reduce((acc, chunk) => {
+    const filteredLines = chunk.changes.filter(change => change.type === 'add')
+    return acc.concat(filteredLines)
+  }, [])
```

Comment:
We have the same code in `hasAddedLinesAfterVersionInChangelog()`, can we extract it to `getAddedLinesInChangelog()`?